packages/terra-collapsible-menu-view/src/terra-dev-site/doc/collapsible-menu-view/CollapsibleMenuView.1.doc.mdx

```diff
@@ -39,12 +39,6 @@ This component requires the following peer dependencies be installed in your app
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-## Usage
-
-```jsx
-import CollapsibleMenuView from 'terra-collapsible-menu-view';
```

Comment:
We should actually keep this to be consistent with the doc changes so far. I think this add value for a quick reference to see how to import the component pieces

Reply:
Added back f75db9b